### PR TITLE
Fix for "open in colab" opens the same zip after updates

### LINKED ISSUE #291: "Open in Colab" opens the same zip even if we update the config in UI
### Steps to reproduce the bug

- Select a template
- Click on "Open in Colab"
- Open Training tab and select Fire in arg parser
- Click on "Open in Colab"

Both links will be the same but second is expected to be different.

### PR DESCRIPTION
**Pull request by theory-in-progress**
### Description

Fix #291 

---

### What is the purpose of this pull request?

- [x] Bug fix
- [ ] New feature
- [ ] Other

### CODE REVIEW

File: functions/colab.js [hunk 1]
```diff
@@ -25,8 +27,10 @@ exports.handler = async function (event, _) {
   // generate a base64 format for pushing to GitHub
   // with Octokit.
   for (const filename in code) {
+    hash += code[filename]
     zip.file(filename, code[filename])
   }
+  const nbUid = uuidv5(hash, NAMESPACE)
```
Comment (vfdev-5) [inline functions/colab.js:33]: Can we use another example here ?
```
uuidv5('https://www.w3.org/', uuidv5.URL);
```
such that we do not need to define a namespace.

File: src/components/NavColab.vue [hunk 2]
```diff
@@ -111,6 +111,7 @@ export default {
       }
       colabText.value = 'Open in Colab'
     }
+    // watch(store.config, () => downloadProject)
```
Comment (vfdev-5) [inline src/components/NavColab.vue:114]: Do we need this ? In general we should remove commented code or explain why it is in the codebase

Comment (theory-in-progress) [reply src/components/NavColab.vue:114]: Ahh, It's not required there. Didn't mean to commit that change...

**Comment by vfdev-5** [comment]
Have you tested on the deployed PR if it works ?

**Comment by vfdev-5** [comment]
For permanent code link (for wget) it should work the same way ?

**Comment by theory-in-progress** [comment]
Yes, was doing that exactly. For now the problem of opening in colab is fixed. I will send a new PR for that?

**Review by vfdev-5 — APPROVED** [review]
Thanks for the fix @theory-in-progress !

**Pull request merged**